addons/mail/static/src/discuss/core/public/welcome_page.js

```diff
@@ -34,6 +40,14 @@ export class WelcomePage extends Component {
         if (this.store.guest) {
             await this.personaService.updateGuestName(this.store.self, this.state.userName.trim());
         }
+        browser.localStorage.setItem(
+            "discuss_call_preview_join_mute",
+            Boolean(!this.state.audioStream)
```

Comment:
```suggestion
            !Boolean(this.state.audioStream)
```

Reply:
Then the boolean is not required at all :) `!this.state.audioStream` will already be a bool.

Reply:
it even saves lines as it was taking 4!